app/(main)/settings/team/team-members.tsx

```diff
@@ -1,17 +1,43 @@
 import { Card } from "@/app/(main)/settings/components/card";
 import { fetchCurrentTeam, isProPlan } from "@/services/teams";
-import { getCurrentUserRole } from "./actions";
+import { getCurrentUserRole, getTeamMembers } from "./actions";
 import { TeamMembersForm } from "./team-members-form";
 import { TeamMembersList } from "./team-members-list";
 
 export async function TeamMembers() {
 	const team = await fetchCurrentTeam();
 	const { data: teamRole } = await getCurrentUserRole();
 
+	const result = await getTeamMembers();
+	const currentUserRoleResult = await getCurrentUserRole();
```

Comment:
nits:

It seems calling the same method in L9, but is it possible to consolidate it here without calling it again?